README.rst

```diff
@@ -150,6 +150,20 @@ Tune security settings (these are default):
           # Set X-Frame-Options
           frame_options: sameorigin
 
+Tune the CustomLog format:
+
+.. code-block:: yaml
+
+    parameters:
+      apache:
+        server:
+          site:
+            foo:
+              enabled: true
+              type: static
+              custom_log_format: >-
```

Comment:
Maybe this would be more generic and adding possibility to change log file location, etc. or disable at all

```log:
  custom:
    format:
```